packages/opentelemetry/src/utils/parseSpanDescription.ts

```diff
@@ -234,48 +232,31 @@ function getGraphqlOperationNamesFromAttribute(attr: AttributeValue): string {
 }
 
 /** Exported for tests only */
-export function getSanitizedUrl(
+export function getParsedUrl(
   attributes: Attributes,
-  kind: SpanKind,
-): {
-  url: string | undefined;
-  urlPath: string | undefined;
-  query: string | undefined;
-  fragment: string | undefined;
-  hasRoute: boolean;
-} {
+): [parsedUrl: ReturnType<typeof parseStringToURLObject>, httpRoute: string | undefined] {
```

Comment:
can we get the httpRoute logic out of here somehow? It seems like all we do is read the http.route attribute and return it.